ui/desktop/src/main.ts

```diff
@@ -1020,6 +1024,65 @@ ipcMain.handle('get-quit-confirmation-state', () => {
   }
 });
 
+// Handle macOS dictation
```

Comment:
nit: also planning to remove this, this looks like a leftover from a different attempt maybe, it appears unreachable to me at the moment. please let me know if i got that wrong!